app/scripts/views/base.js

```diff
@@ -558,13 +559,17 @@ function (Cocktail, _, Backbone, $, p, AuthErrors,
      * navigate to another screen
      */
     navigate: function (page, options) {
-      options = options || {};
-      if (options.success) {
-        this.ephemeralMessages.set('success', options.success);
-      }
-      if (options.successUnsafe) {
-        this.ephemeralMessages.set('successUnsafe', options.successUnsafe);
+      function setEphemeralMessages(name, value, condition) {
```

Comment:
All three invocations of this function pass the same value for `value` and `condition`. You could just pass in `value` and rename the function something like `setEphemeralMessageIfAvailable` or `setEphemeralMessageIfDefined`